mtp_api/apps/mtp_auth/views.py

```diff
@@ -200,7 +224,7 @@ def post(self, request):
 
             send_email(
                 user.email, 'mtp_auth/reset_password.txt',
-                gettext('Your new Money To Prisoners password'),
+                gettext('Your new Prisoner Money password'),
```

Comment:
this one was me. but it's not like we use "Money To Prisoners" anywhere else ever